src/test/java/address/util/TestSerializationClass.java

```diff
@@ -0,0 +1,73 @@
+package address.util;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * A class used to test serialization and deserialization
+ */
+public class TestSerializationClass {
```

Comment:
This  class is for testing purpose only. Basically this class is used to test serialization and deserialization so that we don't need to use our models for testing the serializers. For example, if we port the serializer to another app, we don't need to rewrite the tests to replace `Addressbook` with another class because the tests depend on this class instead of `Addressbook`.

Of course, we need to write another test to ensure `Addressbook` can be serialized properly, too.